### ISSUE TITLE
Add transform configuration

### ISSUE THREAD

**Issue by hcentelles** [thlorenz/es6ify#75]
I can't find a way to configure (select a filePattern to compile) es6ify while using it with karma-browserify. Exclude deps (node_modules) from compilation is mandatory for me, so I had to fork the repo and change the line that set the regex from `/\.js$/` to `/^(?!.*node_modules)+.+\.js$/`, in order to be able to run my tests.

**Comment by erykpiast** [comment]
I've similar problem, but not only with tests. It's more general - when transformation is applied to the module, `"use strict";` expression (statement? : ) is added to the beginning of the module's code. It obviously breaks libraries that don't work in strict mode, like Chai or Rx, for example.

**Comment by domenic** [comment]
If the third party library you're using to wrap es6ify doesn't allow you to configure it, that's a bug in that library, not in es6ify.

@erykpiast all ES6 module code runs in strict mode, so when you transpile it into ES5, that's done by adding `"use strict";`. But yes, in general you should not transpile your dependencies.

**Comment by hcentelles** [comment]
@domenic I disagree with you, in this specific case es6ify right now doesn't provide a friendly interface to be configured by a third party library, but it seems like you guys are working to fix it, PR #63 looks like the solution, do you have a estimation of when this would be available?
